### PR TITLE
Adding features

### PR DESCRIPTION
**Pull request by DEVSOG12**
LGTM :)

### CODE REVIEW

**Comment by DEVSOG12** [comment]
It's Okay

File: Flutter/lib/Services/email_service.dart [hunk 1]
```diff
@@ -4,12 +4,20 @@ import 'package:http/http.dart' as http;
 // import 'package:dart_ipify/dart_ipify.dart'; // Later
 
 class EmailService {
-  Future<http.Response> sendtrans(String message, String email, String name) {
-    return http.post(
-      'https://sdgfortb.herokuapp.com/emailtrans',
-      body: jsonEncode(
-          <String, dynamic>{"email": email, "name": name, "message": message}),
-    );
+  Future<http.Response> sendtrans(String metadata, String email, String name,
```
Comment (Devjay1) [inline Flutter/lib/Services/email_service.dart:7]: I See

File: Flutter/lib/Services/firestore_service.dart [hunk 2]
```diff
@@ -34,7 +34,9 @@ class FirestoreService {
       String description,
       String deviceInfo) async {
     try {
-      var qs = await _paymentCollectionReference.doc(uid).get();
+      var qs = await _paymentCollectionReference
+          .doc("Tj5K5VWuyvhjwKO5GmJQGyUCr3i2")
+          .get();
```
Comment (Devjay1) [inline Flutter/lib/Services/firestore_service.dart:39]: Remove Hardcoded uid

**Review by Devjay1 — APPROVED** [review]
Let Me Let This one go 

Remember to change the UID to `user.uid`

File: Flutter/lib/Services/firestore_service.dart (hunk 2 above)
Comment (DEVSOG12) [reply Flutter/lib/Services/firestore_service.dart:39]: Oh

File: Flutter/lib/Services/email_service.dart (hunk 1 above)
Comment (DEVSOG12) [reply Flutter/lib/Services/email_service.dart:7]: Yh 
The Problem was I was passing `<String, dynamic>` instead of a normal map

**Pull request merged**